File: Math, Logical Implememt/GCD, LCM/acmicpc.net/1934/Main.js
"use strict";

const inputFilePath = "/sample.txt";
const input = require("fs")
  .readFileSync(__dirname + inputFilePath)
  .toString()
  .trim()
  .split("\n");
// const input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n");

const T = Number(input[0]);
for (let i = 1; i <= T; i++) {
  const [A, B] = input[i].split(" ").map((x) => Number(x));
  let [a, b] = [A, B];
  while (b != 0) {
    const r = a % b;
    a = b;
    b = r;
  }
  const g = a;
  const l = g * (A / g) * (B / g);
  console.log(l);
}

File: Math, Logical Implememt/GCD, LCM/acmicpc.net/1934/Main.py
# import sys
# input = sys.stdin.readline
file = open("./01 ing/acmicpc.net/sample.txt", "r")
input = file.readline
T = int(input())
for i in range(T):
    A, B = list(map(int, input().split(" ")))
    a, b = A, B
    while(b != 0):
        r = a % b
        a = b
        b = r
    g = a
    l = int(g*(A/g)*(B/g))
    print(l)
